.cloud/.azure/test/test_aks_deploy.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "dev-aks",
+    "deployment_compute_target": "devAKS",
```

Comment:
I think this compute target (aks cluster name) does not exist here. Use the same one as in aml-run.

Reply:
you are right I will correct it